Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -286,6 +294,17 @@ open class NavigationMapView: UIView {
         mapView.addGestureRecognizer(mapTapGesture)
     }
     
+    public func setupPredictiveCaching(_ predictiveCacheOptions: PredictiveCacheOptions) {
```

Comment:
This public method needs a documentation comment. `predictiveCacheOptions` isn’t a “predictive caching”, so there should be a label on this argument. Since the options argument is required, this method will always enable predictive caching, so we can call it `enablePredictiveCaching(options:)`.